iOS Example Frame/iOS Example Frame/CustomStyle1.swift

```diff
@@ -168,6 +197,7 @@ private struct StyleOrganiser {
     }
 
     struct TextFieldStyle: ElementTextFieldStyle {
+        var textAlignment: NSTextAlignment = .natural
```

Comment:
```swift
 open var textAlignment: NSTextAlignment // default is NSLeftTextAlignment
```

```suggestion
        var textAlignment: NSTextAlignment = .left
```

Reply:
We are support RTL.
_NSTextAlignment.natural
Text uses the default alignment for the current localization of the app._
_The default alignment for left-to-right scripts is NSTextAlignment.left, and the default alignment for right-to-left scripts is NSTextAlignment.right._
https://developer.apple.com/documentation/uikit/nstextalignment/natural

Reply:
👍  What I'm unsure of then is why the default from Apple is `left`. The line I added for documentation is from UIKit.TextField.textAlignment 🤔 🤷‍♂️